lib/class-wp-rest-image-editor-controller.php

```diff
@@ -120,16 +119,26 @@ public function apply_edits( $request ) {
 			return new WP_Error( 'rest_unknown_attachment', $error, array( 'status' => 404 ) );
 		}
 
+		$supported_types = array( 'image/jpeg', 'image/png', 'image/gif' );
+		$mime_type       = get_post_mime_type( $request['id'] );
```

Comment:
```suggestion
		$mime_type       = get_post_mime_type( $attachment_id );
```

Just for consistency.